test/integration/acceptance/transfer_asset_test.cpp

```diff
@@ -159,13 +141,17 @@ TEST_F(TransferAsset, WithOnlyCanTransferPerm) {
  * @then there is an empty proposal
  */
 TEST_F(TransferAsset, WithOnlyCanReceivePerm) {
-  IntegrationTestFramework()
+  IntegrationTestFramework(1)
```

Comment:
Why don't set `3` and omit to skip proposals and block?

Reply:
If we set it to 3, all proposals should have 3 transactions. Since 4 transactions are sent in the test, we will not be able to easily verify that 4th transaction is stateful invalid without sending another two transactions. 
A possible solution is to set `4`, send all transactions, and verify the status of bad expected transaction by doing query and verifying transaction status.
Unfortunately, there is a number (more than 2 tests mentioned in the review) of tests which may be rewritten by applying logic above. I would consider making a separate task to simplify the assertions in acceptance tests.